added cross-reference from deployments (#4060)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/container-deployment/snippet-step-deploy-podman-certs.adoc

. Execute one of the following commands, depending on the SSL certificate variant.
  Replace `<FQDN>` with your fully qualified domain name of the {productname} Server:
+

--
* Using self-signed certificates provided by {productname}:

+

[source,shell]
----
mgradm install podman <FQDN>
----

> * With importing SSL certificates using third-party SSL certificate flags (the example needs to be completed with the other parameters if needed):

Author: ```suggestion
* With importing SSL certificates using third-party SSL certificate flags (the example can adjusted if not all these certificates are needed):
```

+

[source,shell]
----
mgradm install podman <FQDN> \
  --ssl-ca-intermediate <strings> \
  --ssl-server-cert <string> \

>   --ssl-<name> <string> \

Reviewer: For me this is confusing. What should I enter for <name>? 
If I read the comments, you properly mean another SSL parameter (e.g. CA), or do you mean my name?

> mgradm install podman <FQDN> \
>   --ssl-ca-intermediate <strings> \
>   --ssl-server-cert <string> \
>   --ssl-<name> <string> \
>   ...

Author: ```suggestion
mgradm install podman <FQDN> \
  --ssl-ca-intermediate <strings> \
  --ssl-ca-root <string> \
  --ssl-server-cert <string> \
  --ssl-server-key <string> \
  --ssl-db-ca-intermediate <strings> \
  --ssl-db-ca-root <string> \
  --ssl-db-cert <string> \
  --ssl-db-key <string>
```
